2차/류동준_같은-숫자는-싫어.py

```diff
@@ -0,0 +1,10 @@
+def solution(arr):
+    answer = []
+    for data in arr:
+        if not answer:
+            answer.append(data)
+        else:
+            if not answer[-1] == data:
+                answer.append(data)
```

Comment:
javascript에서만 그런건지는 모르겠는데 보통 if - else 보다는 if / if 형태가 좋을 것 같아요

```suggestion
        if not answer:
            answer.append(data)
            continue

        if not answer[-1] == data:
            answer.append(data)            
```